src/theme-context-provider.tsx

```diff
@@ -33,6 +33,22 @@ import type {Theme, ThemeConfig} from './theme';
  */
 const misticaCss = '<MISTICA_CSS>';
 
+const MisticaCss = () => {
```

Comment:
This moves the VE styles to a `<style>` tag in the `<head>`. This fixes style precedence issues with jss in webapp.

This approach doesn't work with SSR, because the `<style>` is only injected in client side. This solution is transitional until we remove JSS from mistica, but meanwhile if a web needs SSR and mistica, it can load the styles using the `css/mistica.css` file and calling passing the `experimental_disableCssInJs` prop to `ThemeContextProvider` to avoid injecting the `<style>` tag in the client